### DISCUSSION TITLE
Running code terminal position

### DISCUSSION THREAD

**Discussion by ginolegigot** [CRAG666/code_runner.nvim#121] [Q&A]
hello!
i wish the terminal running the code appears at the right of my code window in a vertical split, is it possible ? I have tried in term.position "right" or "left", i dont make it work. Only "bot" "top" are working. Any idea ?

**Comment by CRAG666** [comment]
term.position using vert

**Comment by CRAG666** [comment]
pleace read the README.md

**Comment by ginolegigot** [comment]
Oki thx!
